### DISCUSSION TITLE
Log showing caminfo errors

### DISCUSSION THREAD

**Discussion by java007md** [mrlt8/docker-wyze-bridge#963] [Q&A] [answered]
I have not noticed any functionality issues with the bridge, but I am seeing the following in the logs:

[front-porch-cam-50161] [CONTROL] Attempting to GET: caminfo
[front-porch-cam-50161] [CONTROL] ERROR - Invalid command: topic='caminfo' not found cmd=('caminfo', '')

This is a from one of the (5)  wyze V3 cameras. The errors for the others are the same.

Any ideas on what is causing the errors?  I can just ignore the errors, but it would be more reassuring to resolve the issue if possible.

Thanks for any ideas or suggestions.

**Comment by mrlt8** [comment, marked as answer]
this should be fixed in the latest version.

**Reply by java007md** [reply]
Thanks, looks like that fixed it.
